index.html

```diff
@@ -244,7 +244,25 @@ <h2 class="Frameworks">Skills</h2>
         </div>
       </section>
         </div>
-      
+        <!--Contact Form Begins-->
+
+       
+    <div class="contact-form">
+        <h2>Contact Me</h2>
+        <p>
+          If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it 
+        </p>
+        <form action="https://formspree.io/f/mrgjdedp"
+        method="POST">
+          <input type="text" id="name" name="name" maxlength="30" placeholder="Name" required>><br>
+          <input type="email" id='email' name="email" value="email"  required>><br>
```

Comment:
- [ ] Kindly use placeholders to place text inside an input tag. Setting values to it is not recommended.

Reply:
i did not understand this comment